<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearchDialog</class>
 <widget class="QDialog" name="AdvancedSearchDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search In :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="context"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Find :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="find"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Replace :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="replace"/>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>-1</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="caseSensitive">
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="regularExpressions">
        <property name="text">
         <string>Use Regular Expressions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="filePattern"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>File Pattern :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QPushButton" name="replaceButton">
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QPushButton" name="replaceFindButton">
     <property name="text">
      <string>Replace and Find</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QPushButton" name="replaceAllButton">
     <property name="text">
      <string>Replace All</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="previousButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="nextButton">
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
